data/json/overmap/overmap_special/specials.json

```
@@ -63,9 +60,8 @@
     "id": "Riverside Dwelling",
     "overmaps": [ { "point": [ 0, 0, 0 ], "overmap": "riverside_dwelling_north" } ],
     "locations": [ "water" ],
-    "city_distance": [ -1, 25 ],
-    "city_sizes": [ 0, 20 ],
-    "occurrences": [ 0, 12 ],
+    "city_distance": [ 4, 25 ],
+    "occurrences": [ 0, 6 ],
     "flags": [ "CLASSIC", "WILDERNESS" ]
```

```suggestion
    "city_distance": [ 1, 25 ],
    "occurrences": [ 0, 6 ],
    "flags": [ "CLASSIC" ]
```
This is from my personal observations of New England and, hrm, semi-New England (Philadelphia & surroundings).

---

City distance is *very convieniently* measured from the city center ( I have a hunch that it also gets shifted to match the generated city's center, but I didn't dive into that), so 4 is still on the outskirts of all but the smallest towns. Having a cozy riverside cottage squashed between a bank and a police station felt weirder, but I'm not married to that particular value. The Wilderness tag is what allows it to spawn on a rural world.